lightningd/peer_control.c

```diff
@@ -2454,7 +2454,16 @@ static void custommsg_final(struct custommsg_payload *payload STEALS)
 static void custommsg_payload_serialize(struct custommsg_payload *payload,
 					struct json_stream *stream)
 {
+#ifdef COMPAT_V093
```

Comment:
This is not a COMPAT_V093 thing!  This is a deprecated_apis thing.

COMPAT_V093 means "if you first started this node at 0.9.3, we will still work".  It's mainly for database fields, but can also apply to existing channels with peers.